demisto_sdk/commands/common/docker_helper.py

```diff
@@ -49,6 +49,7 @@
 )
 
 TEST_REQUIREMENTS_DIR = Path(__file__).parent.parent / "lint" / "resources"
+DOCKER_CONTAINER_TIMEOUT = 300
```

Comment:
```suggestion
DOCKER_CONTAINER_TIMEOUT = int(os.getenv("DOCKER_CONTAINER_TIMEOUT")) or 300
```
might not be accurate here.